requirements/base.in

```diff
@@ -2,7 +2,7 @@
 -c constraints.txt
 
 # Django/Framework Packages
-Django>=2.2
+Django<2.3
```

Comment:
This constraint should be moved to `constraints.txt` file now to reflect the change in other requirement files as well.